requirements.txt

```diff
@@ -1,11 +1,7 @@
 praw==4.4.0
 redis<3.0.0
-hiredis
-better_exceptions
 addict
-cython
```

Comment:
Cython is required to compile tesserocr, and tersserocr doesn't always pull it in on its own.